pkg/jx/cmd/create_vault.go

```diff
@@ -174,26 +174,32 @@ func (o *CreateVaultOptions) createVaultGKE(vaultName string) error {
 		o.GKEZone = zone
 	}
 
+	clusterName, err := o.clusterName()
+	if err != nil {
+		return err
+	}
+	log.Infof("Current Cluster: %s\n", util.ColorInfo(clusterName))
```

Comment:
:sadpanda: that this is new code and is using the jx/log package and not logrus :-(

Reply:
OK, not a trivial thing as just changing to `logrus.Infof` gives:
`INFO[0014] Creating GCP service account for Vault backend` rather than the string, which is ugly for user output. Will reconsider for a future PR